tools/batchTestGenerator/internal/testCaseBuilder.go

```diff
@@ -85,7 +85,7 @@ func buildTestCases(runConfig RunConfig) ([]TestCaseInfo, error) {
 						}
 						newTests = append(newTests, newTest)
 					}
-				case "EKS_ARM64":
+				case "EKS_ARM64", "EKS_ADOT_OPERATOR_ARM64":
```

Comment:
This has changed, since operator arm 64 tests do not need the arm64 endpoint/cluster/region variables as it uses the same endpoint/region as the EKS tests, and the cluster is set as a `-var=` flag during `executeTerraformTest.sh` since that takes [precedence](https://unofficialaciguide.com/2021/12/20/terraform-with-cisco-nexus-dashboard-orchestrator-for-building-hybrid-cloud-and-end-to-end-services/#paragraph2-11) over the [`parameters.tfvars`](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/testcases/otlp_metric_adot_operator/parameters.tfvars#L7) used to set the EKS cluster name in a non-arm64 operator test case